tests/Makefile

```diff
@@ -0,0 +1,56 @@
+DOCKER?=docker
```

Comment:
Huh. I'm assuming this just isn't getting used at all right now, and can be deleted. Right @tbrent ?